src/main/app/src/pages/ScheduleActivityPage.js

```diff
@@ -83,6 +45,55 @@ const ScheduleActivityPage = ({isLoggedIn, accessToken, userId}) => {
     updateGuest("");
   }
 
+  const randomFunction = (testData) => {
+
+    // arr that has all the indices at first, but then removes the indices thare taken into consideration.
+    const arr = new Array();
+
+    //populate the array elements as 0 to testData's length. 
+    for (let i = 0; i < testData.length; i++) {
```

Comment:
If the objective is to create an array of indexes of the testData array, you can do: Array.from(Array(testData.length).keys()) or [...Array(testData.length).keys()]

Reply:
Wow, that's great! Will implement it! :D